src/main/java/org/openstreetmap/atlas/geography/atlas/complete/CompleteNode.java

```diff
@@ -243,6 +243,12 @@ public CompleteNode withBoundsExtendedBy(final Rectangle bounds)
         return this;
     }
 
+    @Override
+    public CompleteEntity withGeometry(final Iterable<Location> locations)
+    {
+        return this.withLocation(locations.iterator().next());
```

Comment:
can we throw a check here, just to be safe?

Reply:
What kind of check do you have in mind? A `hasNext()` check that throws a `CoreException` if `false`?

Reply:
@adahn6 I made a change which I think addresses your concerns.

Reply:
that looks great! sorry I should have been more specific-- I realize we'd get an iterator exception somewhere if this element doesn't exist, but I feel like wrapping it in our CoreException makes it easier to debug / find in logs / etc